app/views/govuk_publishing_components/components/docs/intervention.yml

```diff
@@ -6,19 +6,41 @@ body: |
   that would be useful to them. This component would be used to add this personalised content and would
   indicate to the user that this is not normally part of the page, but has been added for them specifically.
 
-  Right now the contents of the component are static, as the MVP of personalised content is only for Start a Business.
-  Since many pages will use this component with the same text, we hard-code it here for now.
+  Parameters:
+
+    - `suggestion_text`: the text to display first, before the link
+    - `suggestion_link_text`: the text of the link
+    - `suggestion_link_text`: the URL of the link
+    - `dismiss_text`: the text of the link used to dismiss the intervention
+
+  The dismiss link will reload the page but the `hide-intervention` query string parameter will cause the back-end not to show the intervention again. Some progressive enhancement will be added in later to avoid reloading the page if JavaScript is available.
```

Comment:
I think this is misleading as the component doesn't actually do this, it relies on backend code being added to support it. It might be better to phrase this more accurately to explain the intent of the dismiss link and that backend code will need to be added into applications to handle this.